spec/features/events.js

```diff
@@ -0,0 +1,21 @@
+import chai from 'chai';
+
+const browser = global.browser; // comes from wdio
+
+/**
+ * @TODO Investigate either setting up CouchDB with some initial events or move
+ *       to a 3rd party CMS/events management system
+ */
+xdescribe('Checking for Events', () => {
```

Comment:
I've marked this test as pending via `xdescribe` as currently when these tests run it doesn't setup the events database so it gets an error when going to this page.

Ideally once we've investigated switching to a 3rd party CMS/events management system, this can be turned back on. 😁